parse.y

```diff
@@ -6112,11 +6112,7 @@ regexp		: tREGEXP_BEG regexp_contents tREGEXP_END
                     }
                 ;
 
-words_sep	: ' ' {}
```

Comment:
Another option is defining `words_sep` like below. @nobu Is there any preference?

```c
words_sep : ' '+
          ;
```

Reply:
JFYI: Generated rules are bit different. The difference is that `words_sep` rule exists or not.

## 1. Keep `words_sep` rule

```
nonempty_list_' ': ' '
                 | nonempty_list_' ' ' '

words_sep: nonempty_list_' '

words: "word list" words_sep word_list "terminator"
```

## 2. Remove `words_sep` rule

```
nonempty_list_' ': ' '
                 | nonempty_list_' ' ' '

words: "word list" nonempty_list_' ' word_list "terminator"
```

Reply:
Thank you, it seems fine to expand `words_sep` to `' '+`.